### ISSUE TITLE
FLUX fill problem

### ISSUE THREAD

**Issue by Moyhub** [black-forest-labs/flux#506]
I used the official example and got very poor image quality.
 code:
import torch
from diffusers import FluxFillPipeline
from diffusers.utils import load_image

image = load_image("https://huggingface.co/datasets/diffusers/diffusers-images-docs/resolve/main/cup.png")
mask = load_image("https://huggingface.co/datasets/diffusers/diffusers-images-docs/resolve/main/cup_mask.png")

pipe = FluxFillPipeline.from_pretrained("black-forest-labs/FLUX.1-Fill-dev", torch_dtype=torch.bfloat16).to("cuda")
image = pipe(
    prompt="a white paper cup",
    image=image,
    mask_image=mask,
    height=1632,
    width=1232,
    guidance_scale=30,
    num_inference_steps=50,
    max_sequence_length=512,
    generator=torch.Generator("cpu").manual_seed(0)
).images[0]
image.save(f"flux-fill-dev.png")

gen image
<img width="1232" height="1632" alt="Image" src="https://github.com/user-attachments/assets/3aef6a11-8076-40c8-885f-aae789789c3c" />
